app/routes/events/EventEditRoute.ts

```diff
@@ -29,10 +30,13 @@ import {
 } from './utils';
 
 const mapStateToProps = (state, props) => {
-  const eventId = props.match.params.eventId;
-  const event = selectEventById(state, {
-    eventId,
-  });
+  const eventIDorSlug = props.match.params.eventId;
```

Comment:
I feel like this should be `eventIdOrSlug`. With `eventIDorSlug` I read it as "event i dor slug".